aio/content/examples/getting-started/src/app/shipping/shipping.component.ts

```diff
@@ -11,15 +12,18 @@ import { CartService } from '../cart.service';
   styleUrls: ['./shipping.component.css']
 })
 // #docregion props
-export class ShippingComponent {
+export class ShippingComponent implements OnInit {
 
-  shippingCosts = this.cartService.getShippingPrices();
+  shippingCosts: Observable<{ type: string, price: number }[]>;
 // #enddocregion props
 
 // #docregion inject-cart-service
   constructor(private cartService: CartService) { }
 // #enddocregion inject-cart-service
 
-
 // #docregion props
+   ngOnInit(): void {
+    this.shippingCosts =  this.cartService.getShippingPrices();
+   }
```

Comment:
Incorrect indentation:

```suggestion
  }
```